aidial_adapter_vertexai/chat/gemini/adapter.py

```diff
@@ -111,6 +114,10 @@ async def parse_prompt(
                 return await Gemini_1_5_Pro_Prompt.parse(
                     self.file_storage, tools, messages
                 )
+            case ChatCompletionDeployment.GEMINI_FLASH_1_5:
```

Comment:
No need to complicate the code by introduction of the alias class `Gemini_1_5_Flash_Prompt`.

```python
            case (
                ChatCompletionDeployment.GEMINI_PRO_1_5
                | ChatCompletionDeployment.GEMINI_1_5_FLASH
            ):
                return await Gemini_1_5_Pro_Prompt.parse(
                    self.file_storage, tools, messages
                )
```

Reply:
Ok, maybe then let's rename this prompt parser to something like `Gemini_1_5_Prompt`?

Reply:
Fixed, renamed `Gemini_1_5_Pro_Prompt` to `Gemini_1_5_Prompt`